addons/website_slides/static/src/xml/slide_quiz.xml

```diff
@@ -119,6 +119,16 @@
                     </button>
                 </div>
             </div>
+            <div class="mt-4 o_wslides_js_lesson_quiz_resourceinfo d-none">
+                <div  t-if="widget.quiz.slideResources?.length > 0">
+                    <span>Need help? Review related content:</span> 
+                    <ul class="o_wslides_additional_resources" t-foreach="widget.quiz.slideResources" t-as="resource">
```

Comment:
Class is probably not necessary, if css targets the "ul" inide the o_wslides_js_lesson_quiz_resourceinfo div.